### ISSUE TITLE
How to change PDD parameters

### ISSUE THREAD

**Issue by koheiHasegawa2** [asselapathirana/epanettools#36]
Hi,

I could use the PDD analysis using epanettools and pinpoint important pipes in water distribution systems.

For further analysis, I want to change PDD parameters such as threshold pressures and allowable convergence error.

How can I change these parameters in epanettools?

Thank you

**Comment by koheiHasegawa2** [comment]
Postscript:
I have read your paper that describes EPANET-emitter, at WDSA 2010.
According to that paper, the lowest allowable pressure is set at zero and threshold pressure (called adequate pressure) is indexed P_ECUP.

I could found some e_cup variables in some source code files. 
But still, I do not know how to change parameters.

**Comment by mxfeinberg** [comment]
@koheiHasegawa2 any luck on this?  Did you try looking at the [EPANet source code](https://github.com/OpenWaterAnalytics/EPANET) and making the changes there?

**Comment by koheiHasegawa2** [comment]
@mxfeinberg No, I did. I changed my strategy; I am using [WNTR](https://github.com/USEPA/WNTR) to integrate PDD analysis into my program. WNTR's PDD can change the nodal threshold pressures.
